test/index.spec.js

```diff
@@ -165,8 +163,8 @@ describe('lintStaged', () => {
       "
       LOG Running lint-staged with the following config:
       LOG {
-        '*.css': filenames => \`echo \${filenames.join(' ')}\`,
-        '*.js': filenames => filenames.map(filename => \`echo \${filename}\`)
+        '*.css': [Function: *.css],
+        '*.js': [Function: *.js]
```

Comment:
Unfortunately `stringify-object` seemed to break, so I had to replace it with `object-inspect`. This means we lose the ability to nicely print the values of function tasks.